server_admin/topics/users/con-user-impersonation.adoc

```diff
@@ -16,7 +16,12 @@ image:{project_images}/user-details.png[]
 
 In both instances, the *User Account Management* page of the impersonated user is displayed.
 
+ifeval::[{project_community}==true]
+You can access the *Impersonate* from the *Action* menu tab on the *User details* page.
+endif::[]
+ifeval::[{project_product}==true]
 You can access the *Impersonate* button from the *Details* tab on the *Users* page.
+endif::[]
```

Comment:
```suggestion
.Procedure
. Click *Users* in the menu.
. Click a user to impersonate.
ifeval::[{project_community}==true]
. From the *Actions* list, select *Impersonate*.
endif::[]
ifeval::[{project_product}==true]
Click *Impersonate*.
endif::[]
```

Reply:
Also, can you move this procedure to line 11?

Also we need this before the screenshot.
.User details